.gitignore

```diff
@@ -1,7 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I highly recommend you to remove the / before your build folder so it will be ignored too.

Reply:
Again, thank you very much! :-D